<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="lAdresseServer">
              <property name="text">
               <string>Adresse</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="adresseServer"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lPortServer">
              <property name="text">
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="portServer"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="lPseudoClient">
              <property name="text">
               <string>Pseudo</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="pseudoClient"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="boutonConnexion">
            <property name="text">
             <string>Connexion</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="boutonDeconnexion">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Deconnexion</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="chat">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="message">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="boutonEnvoyer">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Envoyer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
